apps/sms/views/shared/js/utils.js

```diff
@@ -522,10 +522,34 @@
     params: function(input) {
       var parsed = {};
       input.replace(rparams, function($0, $1, $2) {
+        if ($2 === 'true') {
```

Comment:
I know at least 2 cases: "focusComposer" and "notifyAboutSavedDraft".

I thought the case for numbers was not as compelling because it was easier to convert (and maybe more time consuming + more difficult to get right to find out if an arg is a number ?)